tests/components/lutron_caseta/test_device_trigger.py

```diff
@@ -338,12 +347,13 @@ async def test_validate_trigger_config_no_device(hass, calls, device_reg):
 async def test_validate_trigger_config_unknown_device(hass, calls, device_reg):
     """Test for no press with an unknown device."""
 
-    config_entry_id = await _async_setup_lutron_with_picos(hass, device_reg)
+    config_entry_id = await _async_setup_lutron_with_picos(hass)
     data: LutronCasetaData = hass.data[DOMAIN][config_entry_id]
```

Comment:
Existing issue (this code is rather old), so fine to punt on this, but we shouldn't access `hass.data` in tests.

https://developers.home-assistant.io/docs/development_testing/#writing-tests-for-integrations

Reply:
Ok, I"ll look into this further for a future code improvement